erpnext/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -1125,6 +1128,26 @@ def calculate_total_for_salary_slip_based_on_timesheet(self):
 				self.gross_pay += self.earnings[i].amount
 		self.net_pay = flt(self.gross_pay) - flt(self.total_deduction)
 
+	def compute_year_to_date(self):
+		year_to_date = 0
+		fiscal_year = frappe.get_list('Fiscal Year',
```

Comment:
We have some called "Payroll Period", instead of fiscal year lets use payroll period as that makes more sense